pkg/controller/replication_controller.go

```diff
@@ -209,6 +211,14 @@ func (rm *ReplicationManager) getPodControllers(pod *api.Pod) *api.ReplicationCo
 		glog.V(4).Infof("No controllers found for pod %v, replication manager will avoid syncing", pod.Name)
 		return nil
 	}
+	if len(controllers) > 1 {
```

Comment:
What happens when the daemon controller overlaps with a replication controller?  Or a job controller?

> On Jun 26, 2015, at 7:03 PM, Daniel Smith <notifications@github.com> wrote:
> 
> In pkg/controller/replication_controller.go:
> 
> > @@ -209,6 +211,14 @@ func (rm *ReplicationManager) getPodControllers(pod *api.Pod) *api.ReplicationCo
> >  		glog.V(4).Infof("No controllers found for pod %v, replication manager will avoid syncing", pod.Name)
> >  		return nil
> >  	}
> > +	if len(controllers) > 1 {
> It's user error to have simultaneous overlapping RCs. Eventually we'll want RC manager to put offending RCs into a "disabled" state, with a "overlaps with RC foo" status message.
> 
> For now, just always using the oldest one seems like a good strategy.
> 
> —
> Reply to this email directly or view it on GitHub.
>